Firestore/Source/API/FIRAggregateField+Internal.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "FIRAggregateField.h"
+#import "FIRFieldPath.h"
+
+#include <string>
+#include "Firestore/core/src/model/aggregate_alias.h"
+#include "Firestore/core/src/model/aggregate_field.h"
+
+namespace model = firebase::firestore::model;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FIRAggregateField (Internal)
+@property(nonatomic, strong, readonly) FIRFieldPath *fieldPath;
+- (model::AggregateField)createInternalValue;
+- (model::AggregateAlias)createAlias;
+- (const std::string)name;
+@end
```

Comment:
I think this is better:

```
@interface FIRAggregateField (Internal)
- (FIRFieldPath *)fieldPath;
- (model::AggregateField)createInternalValue;
- (model::AggregateAlias)createAlias;
- (const std::string)name;
@end
```

(removing the readonly `@property` and adding an accessor instead)